node/src/main/scala/com/wavesplatform/state/Blockchain.scala

```diff
@@ -39,8 +39,8 @@ trait Blockchain {
 
   def wavesAmount(height: Int): BigInt
 
-  def transferById(id: ByteStr): Option[(Int, TransferTransaction)]
-  def transactionInfo(id: ByteStr): Option[(Int, Transaction)]
+  def transferById(id: ByteStr): Option[(Int, TransferTransaction, Boolean)]
```

Comment:
It will return only succeed transactions: 
https://github.com/wavesplatform/Waves/blob/019738ba4977fc0223ab85a7dc1bb741098181b3/node/src/main/scala/com/wavesplatform/transaction/smart/WavesEnvironment.scala#L45